docs/source/plugins/index.rst

```diff
@@ -146,10 +146,58 @@ Regardless of operator type, all operators can be executed from code. For exampl
 Placements (Menus and Options)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The following images and titles correspond to the available menu ids. MenuItem operators are appended to the specified menu id.
+Operator placement provides a mechanism to add a visual element in the FiftyOne
+associated to an operator. As of right now, operator only supports button
+placement that can invoke the associated operator on click. Below are list of
+places you can display your operator placement in the app:
+
+  - SAMPLES_GRID_ACTIONS
+  - SAMPLES_GRID_SECONDARY_ACTIONS
+  - SAMPLES_VIEWER_ACTIONS
+  - EMBEDDINGS_ACTIONS
+  - HISTOGRAM_ACTIONS
+  - MAP_ACTIONS
+  - MAP_SECONDARY_ACTIONS
+  - DISPLAY_OPTIONS
+
+
+To add a placement for an operator to one of these places in the app, your
+operator class much implement method `resolve_placement` as demonstrated below:
+
+**Python Operator**
+.. code-block:: python
+
+    def resolve_placement(self, ctx):
```

Comment:
@brimoor is there a way to show multiple code examples with a tab to select the language?

Reply:
yeah @imanjra you can use `.. tabs::`: https://github.com/voxel51/fiftyone/blob/660545dcbc4ad3883cc9bb59f53c212b5e85e628/docs/source/user_guide/brain.rst?plain=1#L536

Reply:
Nice! Let me give it a try